includes/classes/ia.core.cache.php

```diff
@@ -243,14 +243,15 @@ protected function _createJsFile($file, $type = 'config')
 
                 // get list of languages
                 $languagesList = $iaDb->assoc(
-                    ['code', 'title', 'direction', 'flagicon', 'iso' => 'code'],
+                    ['code', 'title', 'direction', 'flagicon', 'iso' => 'code', 'locale'],
                     ('admin_lang' == $type ? null : "`status` = 'active'"),
-                    'languages'
+                    iaLanguage::getLanguagesTable()
```

Comment:
Please rest. Probably it's used on installation and no iaLanguage might be defined there.

1. Installation
2. Basic use cases

It looks like it might be the case why it's still not been migrated.

Reply:
Checked installation, API calls and all usages of that. Really works fine.